ibis/backends/tests/test_window.py

```diff
@@ -44,6 +44,55 @@
     HiveServer2Error = None
 
 
+# adapted from https://gist.github.com/xmnlab/2c1f93df1a6c6bde4e32c8579117e9cc
+def ntile(x, bucket: int):
```

Comment:
To not confuse this pandas implementation with `ibis.ntile` in the testing code below, would you mind renaming this to something like `pandas_ntile`?

Reply:
Yep!

Reply:
Done